src/components/organisms/AuthenticationModal/InitialForm/InitialForm.scss

```diff
@@ -16,13 +16,16 @@
     text-align: center;
   }
 
+  .login-text {
+    text-align: center;
+  }
+
   .login-button {
-    padding: $spacing--lg $spacing--xl;
-    box-shadow: inset 0 0 0 3px $primary !important;
     color: white !important;
+    text-decoration: underline;
 
     &:hover {
-      background-color: darken($color: $primary, $amount: 20%);
+      cursor: pointer;
     }
```

Comment:
Why are we needing to add `cursor: pointer;` here?

Reply:
So that it's obvious that you can click on the login text- otherwise it turns into the text cursor and it looks unclickable